packages/medusa/src/strategies/cart-completion.ts

```diff
@@ -14,8 +14,14 @@ import {
   AbstractCartCompletionStrategy,
   CartCompletionResponse,
 } from "../interfaces"
+import {
```

Comment:
thought (this-file): this flow is great for @carlos-r-l-rodrigues' transaction orchestrator.